internal/options/options.go

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright 2020 Red Hat, Inc. All rights reserved.
```

Comment:
I will update that

Reply:
Giving it another look, I found that most projects are using the block comment instead of the one line comment. It is more convenient to use block comments in this case wdyt?

Reply:
Up to you, like I said its a nit.